pkg/asset/machines/worker.go

```diff
@@ -146,6 +146,14 @@ func defaultVSphereMachinePoolPlatform() vspheretypes.MachinePool {
 	}
 }
 
+func defaultKubevirtMachinePoolPlatform() kubevirttypes.MachinePool {
```

Comment:
Why is the default node size need to be per platform?
I would expect that OpenShift nodes of the tenant cluster run the same software and will have the same requirement.

Reply:
The size parameters can be different per platform, for example GCP has the `InstanceType` parameter
Therefore there should be specific function for each platform

Reply:
still...

Reply:
If all the platforms do the same then I guess it's OK